webpack/webpack.config.js

```diff
@@ -0,0 +1,37 @@
+const path = require('path');
+
+module.exports = {
+  mode: process.env.NODE_ENV === 'production' ? 'production' : 'development',
+  entry: ['./js/main.js', './sass/main.scss'],
+  output: {
+    path: path.resolve(__dirname, '../assets/static/gen'),
+    filename: 'script.js',
+  },
+  module: {
+    rules: [
+      {
+        test: /\.scss$/,
+        use: [
+          {
+            loader: 'file-loader',
+            options: {
+              name: 'style.css',
+            }
+          },
+          {
+            loader: 'extract-loader'
+          },
+          {
+            loader: 'css-loader?-url'
+          },
+          {
+            loader: 'postcss-loader'
+          },
+          {
+            loader: 'sass-loader'
+          }
+        ]
+      }
+    ]
+  }
+};
```

Comment:
And here

Reply:
[https://github.com/creativecommons/creativecommons.github.io-source/pull/31/commits/a48e53bab9ce7f2531c6f3dce5d9ce5c61706f8b](url)